restcomm.android.sdk/src/main/java/org/restcomm/android/sdk/RCDevice.java

```diff
@@ -1128,8 +1128,8 @@ public void onCloseReply(String jobId, RCClient.ErrorCodes status, String text)
    {
       RCLogger.i(TAG, "onCloseReply(): id: " + jobId + ", status: " + status + ", text: " + text);
 
-      if (listener != null) {
-         listener.onReleased(this, status.ordinal(), text);
+      if (this.listener != null) {
```

Comment:
Lost you, can you elaborate why we need to add this. in all these occurrences?

Reply:
in RCDevice, we have in some places listener and in some places this.listener. I just want it to make even.